glusterd2/volgen2/clustergraph.go

```diff
@@ -46,6 +46,23 @@ func clusterGraph(volfile *Volfile, dht *Entry, vol *volume.Volinfo, nodeid uuid
 			}
 		}
 
+		// If Graph is required only for Local Bricks then
```

Comment:
to be cleaner can we use continue with LABEL like below 

```
LABEL:
if filters != nil && filters.onlyLocalBricks {
			
	for _, b := range subvol.Bricks {
			if uuid.Equal(b.PeerID, nodeid) {
				continue LABEL
			}
         }
}
```

Reply:
nice feature, but in this case not much useful. Your example will go wrong and loop will be skipped if one local brick found

Reply:
@aravindavk agreed